README.md

```diff
@@ -225,3 +225,82 @@ Example:
     ```
     pytest -s ecs/mxnet/training/test_ecs_mxnet_training.py::test_ecs_mxnet_training_dgl_cpu
     ```
+
+7. To run sagemaker Local tests, Launch a cpu or gpu instance based with latest deep learning AMI.
+    a) clone your github branch with changes and run the following commands
+       ```
+       git clone https://github.com/(github_account_id)/deep-learning-containers/
+       cd deep-learning-containers && git checkout (branch_name)
+       ```
+    b) login into the ecr repo where the new docker images build are present
+       ```
+       $(aws ecr get-login --no-include-email --registry-ids 669063966089 --region us-west-2)
```

Comment:
would check on ECR documentation, but this may no longer be the preferred way to authenticate to ECR